articles/azure-app-configuration/howto-import-export-data.md

```diff
@@ -456,9 +456,9 @@ From the Azure portal, follow these steps:
     | Parameter          | Description                                                                                                                                                                                                                            | Example                   |
     |--------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------|
     | Prefix             | Optional. This prefix will be trimmed from the imported keys. A key prefix is the beginning part of a key. Prefixes can be used to manage groups of keys in a configuration store. Prefix will be ignored for feature flags.           | *TestApp:*                 |
-    | From label         | Select an existing label to restrict your export to key-values with a specific label. If you don't select a label, only key-values with the "No label" label will be exported. See note below.                                         | *prod*                    |
     | At a specific time | Optional. Fill out to import key-values from a specific point in time. This is the point in time of the resource you selected in the previous step. Format: "YYYY-MM-DDThh:mm:ssZ".                                                    | *07/28/2022 12:00:00 AM*  |
-    | Content type       | Optional. Check the box **Override default key-value content types** and select **Key Vault Reference** or **JSON** under **Content type** to state that the imported content consists of a Key Vault reference or a JSON file. | *JSON (application/json)* |
+    | From label         | Select an existing label to restrict your export to key-values with a specific label. If you don't select a label, only key-values with the "No label" label will be exported. See note below.                                         | *prod*                    |
+    | Export as reference | Optional. Check to export key-values to App Service as App Configuration references. This exports a reference to the latest revision of a key-value in App Configuration. [Learn more](../app-service/app-service-configuration-references.md) |
```

Comment:
If we have a specific time, we still export a reference to the key-values which will point to their latest revision. However, this will be only for key-values that existed at the specified time.

Reply:
As discussed offline, there will be no date selection for exporting as reference and it has been explicitly stated that a reference to the latedt revision fo a KV is exported.

Reply:
Please change it to "latest revision of key-value(s)" instead of "latest revision of a key-value"